intellij-plugin-verifier/verifier-intellij/src/main/java/com/jetbrains/pluginverifier/PluginVerifier.kt

```diff
@@ -253,9 +256,13 @@ class PluginVerifier(
       }
     }
 
-    val packageNotFoundProblems = packageToMissingProblems.map { (packageName, missingClasses) ->
-      PackageNotFoundProblem(packageName, missingClasses)
-    }
+    // Kotlin package are supposed to be part of the platform
+    val kotlinPackage = "kotlin\\b(\\..+)?".toRegex()
```

Comment:
To be honest, I can't get it — why is it needed and how connected to another part.

Reply:
I think I had some issues when runnning the tests locally. And I believe IJ is now shipping kotlin ?
https://plugins.jetbrains.com/docs/intellij/using-kotlin.html#incremental-compilation

I can try remove that change